src/actions/performApiDataRequest.ts

```diff
@@ -18,7 +18,7 @@ import { cacheExpired } from '../selectors/cacheExpired';
 import { RequestHandler } from '../request';
 
 export const getRequestProperties = (endpointConfig: ApiDataEndpointConfig, globalConfig: ApiDataGlobalConfig, state: any, body?: any) => {
-    const defaultProperties = { body, headers: {}, method: endpointConfig.method };
+    const defaultProperties = { body, headers: {}, method: endpointConfig.method, autoTrigger: endpointConfig.method === 'GET' ? true : false  };
```

Comment:
I don't think this is supposed to be here? These "defaultProperties" are used as requestProperties (the properties passed to fetch). Maybe we should clarify the name.